src/patternfly/components/DataList/data-list.scss

```diff
@@ -176,6 +182,30 @@
       font-size: var(--pf-c-data-list--m-compact__check--FontSize);
     }
   }
+
+  &.pf-m-truncate {
+    .pf-c-data-list__cell,
+    .pf-c-data-list__text {
+      --pf-c-data-list__cell--MaxWidth: var(--pf-c-data-list__cell--m-truncate--MaxWidth);
+      --pf-c-data-list__cell--Overflow: hidden;
+      --pf-c-data-list__cell--TextOverflow: ellipsis;
+      --pf-c-data-list__cell--WhiteSpace: nowrap;
```

Comment:
What do you think about adding a reasonable min-width when something is truncated so the cell doesn't shrink until no text is visible? The table uses `--pf-c-table--m-truncate--cell--MinWidth: calc(5ch + var(--pf-c-table--cell--PaddingRight) + var(--pf-c-table--cell--PaddingLeft));`

Reply:
I added 20ch, because the data-list cells didn't have any paddingRight or paddingLeft to base the calculation off of, so i thought keeping it to characters would make the most sense right now but let me know if you have different thoughts